src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderPieceService.java

```diff
@@ -161,36 +163,30 @@ public Future<Void> updatePieceRecord(Piece piece, RequestContext requestContext
    * @return CompletableFuture with void.
    */
   public Future<Void> openOrderUpdateInventory(CompositePoLine compPOL, Piece piece, boolean isInstanceMatchingDisabled, RequestContext requestContext) {
-    if (Boolean.TRUE.equals(compPOL.getIsPackage())) {
-      return titlesService.getTitleById(piece.getTitleId(), requestContext)
-        .compose(title -> titlesService.updateTitleWithInstance(title, isInstanceMatchingDisabled, requestContext).map(title::withInstanceId))
-        .compose(title -> {
-          if (piece.getHoldingId() != null) {
-            return Future.succeededFuture(piece.getHoldingId());
-          }
-          if (!PoLineCommonUtil.isHoldingsUpdateRequired(compPOL)) {
-            return Future.succeededFuture();
-          }
-          return inventoryHoldingManager.createHoldingAndReturnId(title.getInstanceId(), piece.getLocationId(), requestContext)
-            .map(holdingId -> {
-              piece.setLocationId(null);
-              piece.setHoldingId(holdingId);
-              return holdingId;
-            });
-        })
-        .compose(holdingId -> {
-          if (PoLineCommonUtil.isItemsUpdateRequired(compPOL)) {
-            return inventoryItemManager.openOrderCreateItemRecord(compPOL, holdingId, requestContext);
-          }
-          return Future.succeededFuture();
-        })
-        .onSuccess(itemId -> Optional.ofNullable(itemId).ifPresent(piece::withItemId))
-        .mapEmpty();
-    }
-    else
-    {
+    if (!Boolean.TRUE.equals(compPOL.getIsPackage())) {
```

Comment:
Boolean.FALSE.equals(compPOL.getIsPackage() is more readable from my prespective.

Reply:
if you are chaning, please change in other places of pr

Reply:
I think if we use compared to FALSE and the getIsPackage() is null, then it will bypass the check, so comparing to TRUE eliminates the need to check for nulls, as both false and null would give same result when using TRUE. What do you think about this?

Reply:
Basically it can be a dangerous swap because boolean flags can be null in our modules.
But for this module we are okay because we have default value declared as false. 

![image](https://github.com/folio-org/mod-orders/assets/25097693/5049cd89-1aae-402d-93cd-3b6bc3a0011a)

Reply:
I see that this field isPackage added in 2020.
Interesting what it will be for orders created before 2020. Will it be automatically populated as false
![image](https://github.com/folio-org/mod-orders/assets/25097693/390b9a6e-ffdb-44e5-8c40-b5a6baace1c7)

Reply:
@SerhiiNosko Thanks for the heads up, changed to FALSE

Reply:
So if some onging order created in 2019 and will have  this check now: Boolean.FALSE.equals(compPOL.getIsPackage()) - it will return null and we will consider this order as non package that we extect, so I think we can use this check

Reply:
@Saba-Zedginidze-EPAM thanks for explanation, never thought about before, You can leave as it was.

Reply:
@SerhiiNosko but would it not match with the current schema and populate with default values if it is missing?

Reply:
to be on safe side lets revert to initial state

Reply:
Got it, reverted